R/fonts.R

```diff
@@ -0,0 +1,72 @@
+#' Check font aliases for missing families and faces
+#' 
+#' @param aliases A list of families of faces. Typically created with
+#'   \code{fontquiver::font_families()}.
+#' @export
+check_aliases <- function(aliases) {
```

Comment:
this is meant only for end users, to check what aliases are not covered when they supply fonts created with `fontquiver::font_families()`. I thought this'd be handy since fonts don't cover all combinations of families and faces. Should I remove this?